plugins/generator-1.16.4/forge-1.16.4/generator.yaml

```diff
@@ -1,5 +1,5 @@
 name: Minecraft Forge for @minecraft (@buildfileversion)
-basefeatures: [variables]
+basefeatures: [variables, model_json, model_obj]

```

Comment:
Remove this, for now, this will be checked once all mod elements are tested to work with this.